staging/src/k8s.io/apimachinery/pkg/util/json/json.go

```diff
@@ -66,11 +66,36 @@ func Unmarshal(data []byte, v interface{}) error {
 		// If the decode succeeds, post-process the map to convert json.Number objects to int64 or float64
 		return convertSliceNumbers(*v, 0)
 
+	case *interface{}:
+		// Build a decoder from the given data
+		decoder := json.NewDecoder(bytes.NewBuffer(data))
+		// Preserve numbers, rather than casting to float64 automatically
+		decoder.UseNumber()
+		// Run the decode
```

Comment:
nit: comments don't add much, just the second one

Reply:
was a copy/paste from the other stanzas, I don't feel that strongly